examples/PingPongHPdio0/PingPongHPdio0.ino

```diff
@@ -0,0 +1,227 @@
+/*
+ *  Copyright (c) 2014, Ivor Wanders
+ *  Copyright (c) 2018, Charles Taylor
+ *  
+ *  MIT License, see the LICENSE.md file in the root folder.
+*/
+
+// Set up and tested with the AdaFruit Feather M0 RFM69HCW modules.
```

Comment:
Cool, thanks for adding this example.

I think it's best to clearly state in this example that it shows how to use the lib with DIO0 connections, as that's the main purpose, perhaps add a sentence on how it works around the difference between DIO2 in the other examples and DIO0. (checking the different events, which still work :+1: )

Reply:
These are all good suggestions. I'm going to close this pull request and submit a more pure patch.